docs/06-troubleshooting/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Troubleshooting
```

Comment:
**nitpick:** I think the order in the menu isn't intuitive

**suggestion:** move it above the `Upgrade` section
![2024-04-30_09-33](https://github.com/front-commerce/developers.front-commerce.com/assets/75968/9c296186-f4e3-45ec-a022-c76eae95d438)